packages/app-extensions/src/actions/modules/actionHandlers/legacyAction.js

```diff
@@ -234,7 +239,13 @@ export default function* (definition, selection, parent, params, config) {
   action.getSelectionNumber = () => selection.type === 'ID' ? selection.ids.length : 0
   action.getSelection = () => legacySelection
 
-  action.perform()
+  try {
+    action.perform()
+  } catch (e) {
+    if (!ignoredExceptionMessages.contains(e)) {
```

Comment:
- Is there a `contains` method on the array? shouldn't it be `includes(str)`?
- usually `Error` objects are thrown which have the message on the `message` property. Maybe we should check if `e` is an instance of `Error` or of `String` and then act accordingly